src/bundle.c

```diff
@@ -158,7 +158,8 @@ int required_by(struct list **reqd_by, const char *bundle_name, struct manifest
 					print_msg = false;
 					info("%s", msg);
 				}
-				info(" - %s\n", name);
+				info(" - ");
```

Comment:
Separate bug fixes or other improvements from main commit. It's hard to understand why this change is made here

Reply:
ok, will do